index.html

```diff
@@ -0,0 +1,146 @@
+          <li><a href=""><img src="images/social/facebook-16.png" alt="facebook"></a></li>
+          <li><a href="#logout">Logout</a></li>
+          <li><a href="#mypage"> My Page</a></li>
+          <li><a href="#Portuguese"> Portuguese</a></li> 
+        </ul>
+      </nav>
+    </header>
+    <header>
+      <div class="logo">
+        <a href="index.html"
+          ><img src="images/Logo/logo.avif" alt="logo"   
+        /></a>
+      </div>
+      <nav class="hamb-menu">
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="about.html">About</a></li>
+          <li><a href="#Contact">News</a></li>
+        </ul>
+      </nav>
+      <button type="button" id="header-button">Collision Campaign</button>
+    </header>
+    <div class="popup-menu">
+      <div class="menu-button">
+        <span class="hamburger"></span>
+        <span class="hamburger"></span>
+        <span class="hamburger"></span>
+      </div>
+      <div>
+        <ul>
+          <li>
+            <a href="index.html">Home</a>
+          </li>
+          <li>
+            <a href="about.html">About</a>
+          </li>
+          <li>
+            <a href="#Contact">News</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+    <main class="main-content">
+      <section id="main">
+        <div>
+          <h1>"Hello! Tech World"</h1>
+          <h2>WEB SUMMIT 2023</h2>
+          <p id="introduction">
+            Although times may be uncertain, one thing
+            remains true. There is a simple power in people
+            coming together.
+          </p>
+          <h3>2023.07.24(MON) ~ 28(FRI)</h3>
+          <p>ENNERCARE CENTRE, TORONTO - CANADA.</p>
+        </div>
+      </section>
+      <section id="programs">
+        <h2>Main Program</h2>
+        <hr class="solid" />
+        <div id="all-programs">
+          <article class="program-cards">
+            <img src="images/programs/program1.png" alt="lecture" />
+            <h2>Lecture</h2>
+            <p>
+              Listen to speakers from various countries about the messages of
+              sharing and opening
+            </p>
+          </article>
+          <article class="program-cards">
+            <img src="images/programs/program2.png" alt="Exhibition" />
+            <h2>Exhibition</h2>
+            <p>
+              Listens to speakers from various countries about the messages of
+              sharing and opening
+            </p>
+          </article>
+          <article class="program-cards">
+            <img src="images/programs/program3.png" alt="Forum" />
+            <h2>Forum</h2>
+            <p>
+              Have the time to share your thoughts and opinions with experts
+              for each topic.
+            </p>
+          </article>
+          <article class="program-cards">
+            <img src="images/programs/program4.png" alt="Workshop" />
+            <h2>Workshop</h2>
+            <p>
```

Comment:
Please ensure that the pages should look almost identical to the original design (the margins should be the same as in the original design and the fonts sizes).

Your Implementation | Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/164412127-14040f71-89ac-4b38-9042-d8a441766dd5.png) | ![image](https://user-images.githubusercontent.com/10332499/164412199-6807afdb-7dbd-4d7b-9d36-d99337662b00.png)

Your Implementation | Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/164412611-a592b4e3-09e1-4f4f-a459-924549865734.png) | ![image](https://user-images.githubusercontent.com/10332499/164412694-e75a3220-9d13-4d50-b762-ec0ec9700ff6.png)

Your Implementation | Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/164412926-23925bc8-9f6f-4b13-948f-91bd95b7d022.png) | ![image](https://user-images.githubusercontent.com/10332499/164413052-94bb4963-3edd-4efa-96d0-d58d3f48fc47.png)

Reply:
resolved